docs/src/components/landing/Demo.js

```diff
@@ -0,0 +1,32 @@
+import * as React from 'react';
+import Box from '@mui/material/Box';
+import dynamic from 'next/dynamic';
+
+const DynamicVideo = dynamic(() => import('./DemoVideo'), {
+  suspense: true,
+});
+
+export default function HeroDemo() {
```

Comment:
Something is off in the way we handle the intermediary loading state: it's 🎄. See the filmstrip: https://www.webpagetest.org/video/compare.php?tests=220823_AiDcK0_869-r:1-c:0

Reply:
Adding a fixed width to the element fixes this: https://www.webpagetest.org/video/compare.php?tests=220824_BiDc9B_85X-r:2-c:0-e:filmstrip